crates/wysiwyg/src/dom/find_range.rs

```diff
@@ -196,6 +204,26 @@ where
     )
 }
 
+fn process_mention_node<S>(
+    node: &MentionNode<S>,
+    start: usize,
+    end: usize,
+    offset: &mut usize,
+) -> Option<DomLocation>
+where
+    S: UnicodeString,
+{
+    // Mentions are like 1-character text nodes
```

Comment:
Slightly worried about this messing up selection conversions, because it will make `replace_all` ranges NOT match the number of actual characters in the HTML. 

Given that I'm pretty sure the way to go is to use a custom HTML tag if possible rather than `a` because clients will need an easy way to know which of those bits of HTML have an actual 1-length rather than what's actually between the tags.